core/makeswift/components/products-carousel/products-carousel.makeswift.tsx

```diff
@@ -116,7 +141,7 @@ runtime.registerComponent(
         }
 
         return {
-          id: title ?? '',
+          id: index.toString(),
```

Comment:
We should use a stable identifier here instead of the index

Reply:
Should we be forcing users to enter an ID or something? Alan was saying using index is the most intuitive way outside of entering one in manually

Reply:
Don't we have the ID of the product?